app/controllers/prison/feedbacks_controller.rb

```diff
@@ -0,0 +1,28 @@
+class Prison::FeedbacksController < ApplicationController
+  before_action :authorize_prison_request
+  before_action :set_inbox_navigation_count, if: -> { current_user }
```

Comment:
Yes, but only worth doing when the dashboards have been rolled out. The prison/visits controller for example loads the counts depending if they are in the trial. I think once it has been rolled out there'll be enough logic to create a dashboard controller super class that has all of the dashboards controller defaults.